### ISSUE TITLE
Accessibility - add alt text to mode icons

### ISSUE THREAD

**Issue by astroash** [FAC-11/3To5#45]
In the summary and journey details there is no alt text for the icons. This could be added to help with accessibility

**Comment by astroash** [comment]
My bad, this had already been raised in #31
